framework/include/constraints/EqualValueEmbededConstraint.h

```diff
@@ -0,0 +1,101 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef EqualValueEmbededConstraint_H
+#define EqualValueEmbededConstraint_H
+
+// MOOSE includes
+#include "NodeElemConstraint.h"
+// #include "ContactMaster.h"
+
+/**
+ * Formulations, currently only supports KINEMATIC and PENALTY
+ */
+enum Formulation
+{
+  KINEMATIC,
+  PENALTY,
+  INVALID
+};
+
+// Forward Declarations
+class EqualValueEmbededConstraint;
+
+template <>
+InputParameters validParams<EqualValueEmbededConstraint>();
+
+/**
+ * A EqualValueEmbededConstraint forces the value of a variable to be the same
+ * on overlapping portion of two blocks
+ */
+class EqualValueEmbededConstraint : public NodeElemConstraint
+{
+public:
+  EqualValueEmbededConstraint(const InputParameters & parameters);
+
+  virtual void timestepSetup() override;
+  virtual void jacobianSetup() override;
+  virtual void residualEnd() override;
+
+  virtual Real computeQpSlaveValue() override;
+
+  virtual Real computeQpResidual(Moose::ConstraintType type) override;
+
+  /**
+   * Computes the jacobian for the current element.
+   */
```

Comment:
Remove this comment describing the method -- we don't include them for any 'override' methods.